### PR TITLE
 Add Pipeline Check to Verify Frontend

### LINKED ISSUE #15: Add Pipeline Check to Verify Frontend
Add a pipeline check to verify frontend changes similar to the backend "Verify Code" check.

### PR DESCRIPTION
**Pull request by yazan-abu-obaideh**
closes #15

### CODE REVIEW

File: .github/workflows/verify.yaml [hunk 1]
```diff
@@ -18,3 +18,16 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Node set-up"
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+      - name: Install dependencies
+        run: cd frontend && npm install
+      - name: Build
+        run: cd frontend && npm run build -- --mode=production
```
Comment (osama-salman99) [inline .github/workflows/verify.yaml:33]: You can check out the [working-directory](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory) keyword

**Review by osama-salman99 — COMMENTED** [review]
Can you please add a more meaningful title to the PR

**Review by osama-salman99 — CHANGES_REQUESTED** [review]
Can you please add a more meaningful title to the PR

File: .github/workflows/verify.yaml (hunk 1 above)
Comment (yazan-abu-obaideh) [reply .github/workflows/verify.yaml]: Good catch, done

**Review by osama-salman99 — APPROVED** [review]

**Pull request merged**